### PR TITLE
fix: error from mktemp command in MacOS.

### PR DESCRIPTION
**Pull request by appleboy**
Error: `Permission denied` from `mktemp` command in MacOS.

```
$ make test-vendor
usage: mktemp [-d] [-q] [-t prefix] [-u] template ...
       mktemp [-d] [-q] [-u] -t prefix
govendor list +unused | tee "/wc-gitea-unused"
tee: /wc-gitea-unused: Permission denied
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,7 +15,13 @@ SOURCES ?= $(shell find . -name "*.go" -type f)
 
 TAGS ?=
 
-TMPDIR := $(shell mktemp -d)
+TMPDIR := $(shell mktemp -d 2>/dev/null || mktemp -d -t 'tempdir')
```
Comment (sapk) [inline Makefile]: Maybe use a "gitea-temp" in place of "tempdir" to be more indicative ?

File: Makefile [hunk 2]
```diff
@@ -22,7 +15,13 @@ SOURCES ?= $(shell find . -name "*.go" -type f)
...
+
+ifeq ($(OS), Windows_NT)
```
Comment (sapk) [inline Makefile]: Any reason to move this ?

**Comment by sapk** [comment]
Few little comments but LGTM

**Comment by ethantkoenig** [comment]
LGTM (aside from @sapk's comments)

**Comment by appleboy** [comment]
@sapk Done. update temp name.

**Pull request merged**